### ISSUE TITLE
Travel Category

### ISSUE THREAD

**Issue by ohheyrj** [2factorauth/twofactorauth#4799]
Not sure if this has been discussed before, but, would there be an appetite to add a 'Travel' category? For example, Expedia provides both transport and accommodation, it would fit into either of these categories but I think 'Travel' would fit better. The same can be said for TripIt, which is currently listed under Transport.
I am happy to implement this change if people think this is a good idea.

**Comment by person1234-ux** [comment]
I agree airlines along with uber and Lyft would fall in this category.
